### PR TITLE
Norfair West heatdata

### PR DESCRIPTION
**Pull request by DIProgan**
Steam over

### CODE REVIEW

File: region/norfair/west.json [hunk 1]
```diff
@@ -765,17 +803,60 @@
...
+                      {"enemyDamage": {
+                        "enemy": 99,
+                        "hits": 1
+                      }}
+                    ]
+                  ]},
+                  {"cost": [
+                    {"heatFrames": 300},
+                    [
+                      {"ammo": {
```
Comment (fdtaillefer) [inline region/norfair/west.json]: Do you think it's worth also looking at alternatives with wave beam?

File: region/norfair/west.json [hunk 2]
```diff
@@ -785,18 +866,60 @@
...
+                        "enemy": 99,
+                        "hits": 1
+                      }}
+                    ]
+                  ]},
+                  {"cost": [
+                    {"heatFrames": 200},
+                    [
+                      {"ammo": {
+                        "type": "PowerBomb",
```
Comment (fdtaillefer) [inline region/norfair/west.json]: Same question here for wave

File: region/norfair/west.json (hunk 1 above)
Comment (DIProgan) [reply region/norfair/west.json]: Absolutely. Tomorow :)

Comment (DIProgan) [reply region/norfair/west.json]: Difference is less than 50f both from top and bottom door. Leaving them unchanged.

Comment (fdtaillefer) [reply region/norfair/west.json]: Alrighty then

**Pull request merged**